prov/verbs/src/verbs_domain.c

```diff
@@ -355,6 +362,34 @@ vrb_domain(struct fid_fabric *fabric, struct fi_info *info,
 	if (ret)
 		goto err3;
 
+	if (!vrb_gl_data.log_async_events)
+		goto skip_async;
+
+	ret = fi_fd_nonblock(_domain->verbs->async_fd);
+	if(ret) {
+		VRB_INFO(FI_LOG_DOMAIN, "Failed to enable async events for %s: %s\n",
+				_domain->info->domain_attr->name, strerror(-errno));
+		goto skip_async;
+	}
+
+	ret = ofi_epoll_create(&_domain->epoll_fd);
```

Comment:
Why is logging adding epoll overhead?

Reply:
I can switch to poll for getting these events on the cq, which will remove the epoll structures, but a poll will still be required. The only existing poll is in the cq_sread path and I don't think that's enough.